Client/src/Components/Login/Login.jsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  TextField,
+} from "@mui/material";
+import MySnackbar from "../Snackbar/MySnackbar";
+import { useDispatch, useSelector } from "react-redux";
+import { useState, useEffect } from "react";
+import { REQUEST_STATUSES } from "../../constants/apiRequestStatus";
+import {
+  loginFormVisibilityChanged,
+  addNewUser,
+  selectLoginError,
+  selectLoginStatus,
+  resetStatus,
+} from "../../slices/loginSlice";
+import "./Login.css";
+
+export default function Login(props) {
+  const handleCloseSnackBar = () =>
+    setSnackbarState((previous) => ({ ...previous, open: false }));
+
+  const [snackbarState, setSnackbarState] = React.useState({
+    open: false,
+    vertical: "top",
+    horizontal: "center",
+    severity: "success",
+    message: "",
+    handleClose: handleCloseSnackBar,
```

Comment:
Why do you store all this in state? Ususualy you need to store in component state data that is managed by this component, in this case, seems, it is only isSnackbarOpen field. Fields like vertical, horizontal are hardcoded and are not changed anywhere, fields message, severity are already stored in Redux store, so you don't need to store them in component state

Reply:
I don't have fields "message" and "severity" in Redux store, because they are related to Snackbar and I haven't created slice for it.